charts/comet-api-v1/templates/deployment.yaml

```diff
@@ -117,3 +120,6 @@ spec:
         - name: tmp
           emptyDir:
             sizeLimit: "1Gi"
+        - name: {{ .Values.serviceAccount.volumeMounts.name }}
+          secret:
+            secretName: {{ include "comet-api.fullname" . }}-service-account
```

Comment:
if the service account is not created, this would fail.

Reply:
4ec8b855c3b53565bccc0746e1312a7c877b0fef